onboard/src/api/signRequest.js

```diff
@@ -0,0 +1,22 @@
+// import axios from "axios";
```

Comment:
Remove commented code unless it is needed.

Reply:
Deleted